src/large_pages/node_large_page.cc

```diff
@@ -282,6 +282,25 @@ bool IsSuperPagesEnabled() {
 }
 #endif
 
+class MmapPointer {
```

Comment:
```suggestion
class MemoryMapPointer {
```

Reply:
Should this also implement `MemoryRetainer`?

Reply:
`MemoryRetainer` is for classes whose allocations and references can be included in a heap snapshot, so it does not apply here.

Reply:
Yeah, I was thinking more for future potential uses but you're right, makes sense to leave out here